cdap-app-templates/cdap-etl/cdap-etl-batch/pom.xml

```diff
@@ -44,6 +44,7 @@
       <groupId>co.cask.cdap</groupId>
       <artifactId>cdap-etl-lib</artifactId>
       <version>${project.version}</version>
+      <scope>test</scope>
```

Comment:
This is temporary, right? When we have the test framework supports adapter and plugin then we can remove this dependency entirely, right?